parking_lot/shr_sexualhealth.txt

```diff
@@ -8,166 +8,116 @@ CodeSystem:		MTH = http://ncimeta.nci.nih.gov
-
-	
-EntryElement:	ContraceptivesUsed
-Concept:		TBD
-Description:	"How frequently the subject and partner uses contraceptives, and if so, what contraceptive methods used."
-Based on:		SocialHistoryObservation
-Value:			CodeableConcept from QualitativeFrequencyVS
-				PanelMembers.EvaluationTopic
-includes 0..*	ContraceptiveMethodUsed
-
-EntryElement:	ContraceptiveMethodUsed
-Based on:		SocialHistoryObservation
-Concept:		MTH#C0700589
-Description:	"A method contraception used (or not) by the subject."
-Value:			CodeableConcept from ContraceptiveMethodVS
+				DataValue from QualitativeFrequencyVS
+
+EntryElement:	ContraceptiveMethod
+Based on:		CodedSocialHistoryObservation
+Description:	"A method contraception used by the subject. This profile can be used to record that a person does or does not use a certain type of contraception, and why."
 				Code is MTH#C0700589
-				EvaluationComponent
+				CodeableConcept from ContraceptiveMethodVS
+				Components.ObservationComponent
 includes 0..1 	ContraceptiveMethodFrequency
-includes 0..*	ContraceptiveMethodReason
-includes 0..*	ContraceptiveNotUsedReason
+includes 0..*	ContraceptiveMethodUsedReason
+includes 0..*	ContraceptiveMethodNotUsedReason
 
 		Element:		ContraceptiveMethodFrequency
-		Based on:		EvaluationComponent
-		Concept:		TBD
+		Based on:		ObservationComponent
 		Description:	"How often the contraceptive method is used by the subject."
-		Value:			CodeableConcept from QualitativeFrequencyVS
+						Code is TBD#TBD
+						DataValue from QualitativeFrequencyVS
 
 		Element:		ContraceptiveMethodReason
-		Based on:		EvaluationComponent
-		Concept:		TBD
+		Based on:		ObservationComponent
 		Description: 	"Reason for using a particular contraceptive."
-		Value:			CodeableConcept from ContraceptiveMethodReasonVS
+						Code is TBD#TBD
+						DataValue from ContraceptiveMethodUsedReasonVS
 						
 		Element:		ContraceptiveNotUsedReason
-		Concept:		TBD
-		Based on:		EvaluationComponent
+		Based on:		ObservationComponent
 		Description:	"If not using a method of contraception, why."
-		Value:			CodeableConcept from ContraceptiveNotUsedReasonVS
+						Code is TBD#TBD
+						DataValue from ContraceptiveMethodNotUsedReasonVS
 
-						
-Abstract Element:	ContraceptiveTopic
-Based on:		MedicationTopic
-Concept:		TBD
-Description:	"A contraceptive for a patient, independent of context of being recommended, used, or not used."
-Value:			CodeableConcept from ContraceptiveMethodVS
+EntryElement:	PregnancyStatus
+Based on:		CodedSocialHistoryObservation
+Description:	"Whether or not the subject is currently pregnant."
+				Code is LNC#82810-3
+				DataValue from YesNoUnknownVS
 
-						
-EntryElement:	ContraceptiveMethodRequested
-Based on:		ContraceptiveTopic
+EntryElement:	PregnancyIntention
+Based on:		CodedSocialHistoryObservation
+Description:	"Whether the subject or his/her partner has a plan or desire to either become pregnant or have a child in the next year/near future."
```

Comment:
The description conflates current and future intentions for pregnancy planning. This conflates LOINC code 64631-5 which only scopes to "current intentions to get pregnant." https://s.details.loinc.org/LOINC/64631-5.html?sections=Comprehensive